src/cases/services/role-exclusions.service.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { RoleExclusion } from '../models/role-exclusions/role-exclusion.model';
+
+@Injectable()
+export class RoleExclusionsService {
+    public static exclusionsUrl = '/api/user/exclusions';
```

Comment:
Would like to see this in a config file

Reply:
We do not have configs for Angular. @paulhowes-HMCTS

Reply:
src/assets/config/config.json?

Reply:
No, thats not meant for that. Its deprecated. In fact we can delete it. @paulhowes-HMCTS

Reply:
Discussed with Uday, this Config is used for the ccd-case-ui-toolkit and not Manage Cases, which is the reason why he has left the exclusionsUrl in here.